xml/ses_glossary.xml

```diff
@@ -52,20 +117,79 @@
     </para>
    </glossdef>
   </glossentry>
-  <glossentry><glossterm>Monitor node, MON</glossterm>
+  <glossentry><glossterm>CRUSH rule</glossterm>
+   <glossdef>
+    <para>
+     The CRUSH data placement rule that applies to a particular pool(s).
```

Comment:
TBH I don't like the a...pools interpretation here but I feel maybe I'm being overly picky.

Reply:
What would you recommend?

Reply:
Hmmm. Either

... applies to particular pools...

or

... applies to a particular pool or pools...

?